cmd/util/cmd/reindex/cmd/results.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+)
+
+func init() {
+	rootCmd.AddCommand(resultsCmd)
+}
+
+var resultsCmd = &cobra.Command{
+	Use:   "results",
+	Short: "reindex sealed result IDs by block ID",
+	Run: func(cmd *cobra.Command, args []string) {
+		db := common.InitStorage(flagDatadir)
+		defer db.Close()
+		storages := common.InitStorages(db)
+		state, err := common.InitProtocolState(db, storages)
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not init protocol state")
+		}
+
+		results := storages.Results
+		blocks := storages.Blocks
+
+		root, err := state.Params().Root()
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not get root header from protocol state")
+		}
+
+		final, err := state.Final().Head()
+		if err != nil {
+			log.Fatal().Err(err).Msg("could not get final header from protocol state")
+		}
+
+		for h := root.Height + 1; h <= final.Height; h++ {
```

Comment:
could we add a log message printing the final height so that we know till what block height was the ER indexed?